packages/shared-ux/button_toolbar/src/buttons/icon_button_group/icon_button_group.styles.ts

```diff
@@ -12,7 +12,7 @@ export const IconButtonGroupStyles = ({ euiTheme }: UseEuiTheme) => {
   return {
     button: {
       '&.euiButtonGroupButton': {
-        backgroundColor: euiTheme.colors.emptyShade,
+        backgroundColor: 'transparent',
```

Comment:
The component seems to be used in a couple of other places. is the change intentional for all the occurrences?

Reply:
This is a good question, @ryankeairns I assumed that this was intentional but was it really?? 😄

Reply:
It was intentional, but I will do a more thorough, visual scan of each usage. I can't think of a time this should not be transparent but, well, you just never know for certain :)

Reply:
Thanx Ryan!

Reply:
Undid this change. It affected Canvas and is quite subjective in nature, so I will just leave it be and not introduce new custom styles for this single button instance.